swift/schema.py

```diff
@@ -1246,3 +1266,15 @@ class SingleValueStmtExpr(Expr):
     An expression that wraps a statement which produces a single value.
     """
     stmt: Stmt | child
+
+class PackExpansionExpr(Expr):
+    """
+    A pattern expression followed by the expansion operator '...'.
+    """
+    pattern_expr: Expr | child
+
+class PackElementExpr(Expr):
```

Comment:
you can now use `@qltest.test_with(PackExpansionExpr)` to avoid having a separate test directory for this, repeating the same sources. You also save on extraction time like this.